apps/web/screens/Grants/SearchResults/SearchResults.tsx

```
@@ -0,0 +1,465 @@
+  const updateUrl = useCallback(() => {
+    if (!searchState) {
+      return
+    }
+    router.replace(
+      {
+        pathname: currentUrl,
+        query: Object.entries(searchState)
+          .filter(([_, value]) => !!value)
+          .reduce(
+            (accumulator, [searchStateKey, searchStateValue]) => ({
+              ...accumulator,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)